examples/property/train/input_torch.json

```diff
@@ -32,7 +32,8 @@
     "fitting_net": {
       "type": "property",
       "intensive": true,
-      "task_dim": 3,
+      "property_name": "band_prop",
```

Comment:
what does "band_prop" stands for?

Reply:
It means the program will read `set.xxx/band_prop.npy` as training data. `band_prop.npy` has humo, lumo and band_gap properties.

Reply:
Can you write your explanation in a README file?

Reply:
I will write a a document in next commit in this PR.

Reply:
I have added document file `doc/model/train-fitting-property.md`